.circleci/config.yml

```diff
@@ -23,17 +27,15 @@ jobs:
     - run: git diff --exit-code
 
   build:
-    executor: golang
+    executor: vm
```

Comment:
Let's stick to the standard docker executor, it's faster to spin up, and this doesn't seem to improve build time.

To improve build time, we'll likely need to break up the build into multiple parallel build instances.